docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,594 @@
-You don't need to set up a trusted cluster just to connect a couple of servers, kubernetes clusters or
-databases behind a firewall. For more information, see how to [join a service to your cluster](../../agents/join-services-to-your-cluster.mdx).
-</Notice>
-
-When a user tries to connect to any resource inside the leaf cluster using the root's proxy
-the connection goes through the reverse tunnel.
-
-This guide will explain how to:
-
-- Add and remove Trusted Clusters using CLI commands.
-- Enable/disable trust between clusters.
+## How it works
+
+In the following example, a managed service provider uses three independent clusters to 
+provide access to clients in different regions:
+
+- Cluster `sso.example.com` is the root cluster. This cluster might have its own resources or be 
+strictly used for collecting audit logs and authenticating users.
+- Clusters `us-east-1a` and `us-east-1b` are two independent clusters in different availability zones.
+- Each cluster is an independent x.509 and SSH certificate authority and can operate autonomously. 
+  However, each leaf cluster can only trust one root cluster at any given time.
+- Each leaf cluster dials back to the root cluster to establish a reverse tunnel. If you have multiple 
+proxies, you have multiple tunnels for high availability.
+
+The following diagram provides a simplified view of the architecture:
+
+![Trusted cluster](../../../img/trusted-clusters/simplified-trusted-cluster.png)
+
+As this diagram suggests, users can sign in to the root cluster to receive certificates signed by the root
+cluster certificate authority. They can then connect to leaf clusters either directly
+or through the root cluster acting as a bastion host.
+After users sign in, they can be recognozied and trusted on the leaf clusters based on informtion in their
+certificate and on how their roles in the root cluster are mapped to roles on the leaf cluster.
+
+The Teleport Auth Service on the leaf cluster determines the resources a user has permissions to access by
+checking the certificate for the following information:
+
+- The list of **principals** that the user is authorized to use. Principals are equivalent to
+  the local logins that have been added to a Teleport user profile.
+- The signature of the **certificate authority** that issued the certificate. In a trusted cluster environment, 
+  the Teleport Auth Service on the root cluster signs the certificate.
+- The **metadata certificate extensions** that are provided by the Teleport Auth Service. Teleport uses 
+  the metadata to store the list of user roles and SSH options like `permit-agent-forwarding`.
+- The expiration date, or **time-to-live** (TTL) to ensure the certificate is still valid.
+
+Based on the information in the certificate, the Teleport Auth Service performs the following actions:
+
+- Verifies that the certificate signature matches one of its trusted clusters.
```

Comment:
This contradicts the statement above. Here we say "one of its trusted [root] clusters" (implying there can be more than one), but above we said "each leaf cluster can only trust one root cluster"

Reply:
This comes from a bullet buried in a detail "How does this work?" under step 4/5 in the current doc. Again, I don't know if it's true, but theoretically, the statement might not be contradictory in that a leaf can only have one root at a time but it does seem that one statement or the other is misleading.

> When a user from the root cluster attempts to access a Node in the leaf cluster, the leaf cluster's Auth Service authenticates the user's certificate and reads these pieces of data from it. It then performs the following actions:

Checks that the certificate signature matches one of its Trusted Clusters.